src/mbgl/map/transform.cpp

```diff
@@ -88,7 +88,7 @@ void Transform::easeTo(const CameraOptions& camera, const AnimationOptions& anim
     double angle = camera.angle ? -*camera.angle * util::DEG2RAD : getAngle();
     double pitch = camera.pitch ? *camera.pitch * util::DEG2RAD : getPitch();
 
-    if (std::isnan(zoom)) {
+    if (std::isnan(zoom) || std::isnan(angle) || std::isnan(pitch)) {
```

Comment:
that's out of this patch scope, however, is it fine we just return here?

Reply:
We could transform this into an assertion, or throw an exception (though I know you're not fond of it) ;) but I'd prefer to treat this in a separate issue/PR.